examples/simple-printer/main.go

```diff
@@ -15,14 +16,25 @@ import (
 // CREATE TABLE ks.tbl (pk int, ck int, v int, PRIMARY KEY (pk, ck)) WITH cdc = {'enabled': 'true'};
 
 func main() {
-	if err := run(context.Background(), []string{"127.0.0.1"}, "local-dc", "ks.tbl"); err != nil {
+	var (
+		keyspace string
+		table    string
+		source   string
+	)
+
+	flag.StringVar(&keyspace, "keyspace", "ks", "keyspace name")
+	flag.StringVar(&table, "table", "tbl", "table name")
+	flag.StringVar(&source, "source", "127.0.0.1", "address of a node in the cluster")
+	flag.Parse()
+
+	if err := run(context.Background(), source, keyspace, table); err != nil {
 		log.Fatal(err)
 	}
 }
 
-func run(ctx context.Context, hosts []string, localDC, tableName string) error {
-	cluster := gocql.NewCluster(hosts...)
-	cluster.PoolConfig.HostSelectionPolicy = gocql.TokenAwareHostPolicy(gocql.DCAwareRoundRobinPolicy(localDC))
+func run(ctx context.Context, source, keyspace, table string) error {
+	cluster := gocql.NewCluster(source)
+	cluster.PoolConfig.HostSelectionPolicy = gocql.TokenAwareHostPolicy(gocql.RoundRobinHostPolicy())
```

Comment:
> extend simple-printer to accept cli arguments 

Hmm, this commit not only adds CLI arguments but also changes the load balancing policy. I think doing that is fine for this simple example, but it should be noted in the cover letter. Alternatively, you can consider keeping the current retry policy but add a parameter for specifying the local DC.